.gitignore

```diff
@@ -41,4 +41,9 @@ testing.dat
 /performanceEvaluation
 /password.txt
 /dependency-reduced-pom.xml
-*.csv
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Hier muss noch korrekt gemerged werden,

Reply:
Das hat mit dem aktuellen PR nichts zu tun. In der Vergangenheit gab es einen Merge-Konflikt, der nicht richtig aufgelöst wurde. Vgl. "<<<<<<< HEAD" etc. Der Merge muss noch korrekt in der Datei durchgeführt werden.